sdk/keyvault/azure-security-keyvault-secrets/vcpkg/vcpkg.json

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) Microsoft Corporation. All rights reserved. 
-# SPDX-License-Identifier: MIT
+# SPDX-License-Identifier: MIT 
```

Comment:
Is this right? Usually trailing whitespace is considered bad. `git diff` flags it by default as an error.

Reply:
i just merged from Azure main , the extra space is gone, good catch though.